app/config.js

```diff
@@ -17,12 +16,12 @@ const winCfg = new Config({
   }
 });
 
-let configDir = homedir();
-if (isDev) {
+let configDir = _paths.homeDir;
+if (_paths.isDev) {
```

Comment:
@chabou The DEV mechanism is part of the Global Keymaps. But for this I need to rework the whole config which is another part. See #1509 with `config/paths.js`